src/Markdig/Helpers/LinkHelper.cs

```diff
@@ -411,7 +411,7 @@ public static bool TryParseInlineLink(ref StringSlice text, out string? link, ou
         {
             // Skip ')'
             text.SkipChar();
-            title ??= string.Empty;
+            // not to normalize nulls
```

Comment:
This comment makes sense if you're looking at this PR, but not on its own. I'd just remove it

Reply:
Changed, explained why doing so in detail.